tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "baseUrl": "/",
     "paths": {
       "*": ["node_modules/*", "src/types/*"]
-    }
+    },
+    "noUnusedLocals": true,
```

Comment:
Curious as to what this does?

Reply:
This makes sure that when the Typescript compiles it checks for unused variables / imports. If there are any it will throw an error and force us to fix